plane/source/docs/qstabilize-mode.rst

```diff
@@ -0,0 +1,78 @@
+.. _qstabilize-mode:
+
+===============
+QSTABILIZE Mode
+===============
+
+QSTABILIZE mode allows you to fly your vehicle manually, but self-levels
+the roll and pitch axis.
+
+.. tip::
+
+   If you're learning to fly, try :ref:`QHOVER <qhover-mode>` or
+   :ref:`QLOITER <qloiter-mode>` instead of
+   QSTABILIZE. You'll have fewer crashes if you don't need to concentrate on
+   too many controls at once.
+
+
+.. warning::
+
+   While QSTABILIZE mode does not necessarily require GPS, switching to QRTL or RTL in case of emergency does. Make sure you do have a reliable
+   position estimate prior to arming, most commonly provided by 3D GPS fix with sufficient HDOP.
+
+
+Overview
+========
+
+-  Pilot's roll and pitch input control the lean angle of the QuadPlane.
+   When the pilot releases the roll and pitch sticks the vehicle
+   automatically levels itself.
+-  Pilot will need to regularly input roll and pitch commands to keep
+   the vehicle in place as it is pushed around by the wind.
+-  Pilot's yaw input controls the rate of change of the heading.  When
+   the pilot releases the yaw stick the vehicle will maintain it's
+   current heading.
+-  Pilot's throttle input controls the average motor speed meaning that
+   constant adjustment of the throttle is required to maintain
+   altitude.  If the pilot puts the throttle completely down the motors
+   will go to their minimum rate (Q_M_SPIN_ARMED) and if the vehicle
+   is flying it will lose attitude control and tumble.
+-  The throttle sent to the motors is automatically adjusted based on
```

Comment:
note that flying backwards in a quadplane is a very bad idea, due to reversal of fixed wing control surfaces leading to instability